recipes/xbpch/meta.yaml

```diff
@@ -0,0 +1,50 @@
+
+{% set name = "xbpch" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "903358ec30cd6ca8c577aa0c9f2b384574f95c525452911b53614c5ae2046271" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - dask
+    - xarray
+
+test:
+  imports:
+    - xbpch
+
+about:
+  home: http://github.com/darothen/xbpch
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'xarray interface for bpch files'
+
+  description: |
+    xpbch is a simple utility for reading the proprietary binary punch
+    format (bpch) outputs used in versions of GEOS-Chem earlier than
+    v11-02. The utility allows a user to load this data into an
+    xarray/dask-powered workflow without necessarily pre-processing the
+    data using GAMAP or IDL.
```

Comment:
Something in the description is crashing the `yaml` parser.
I don't really know what :unamused:

Reply:
Yep it was the lack of spaces :smile: